pkg/apis/sources/v1alpha2/apiserver_conversion_test.go

```diff
@@ -14,15 +14,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package v1alpha2
 
 import (
 	"context"
 	"testing"
 )
 
-func TestApiServerConversionBadType(t *testing.T) {
-	good, bad := &ApiServerSource{}, &ApiServerSource{}
+func TestApiServerSourceConversionBadType(t *testing.T) {
+	good, bad := &ApiServerSource{}, &dummy{}
 
```

Comment:
Don't write additional UT since it already get covered in v1alpha1/apiserver_conversion_test.go